README.md

```diff
@@ -207,3 +207,69 @@ Change the properties below in [pom.xml](./pom.xml) to point to the desired refe
   ...
 </project>
 ```
+
+### Running Integration Tests
+
+#### Pre-Requisites for ITs
+Along with the pre-requisites for [SDK](#pre-requisites) the following are needed. 
+
+* Docker installed 
+  * [Docker Compose](https://docs.docker.com/compose/install/) 
+  * [Docker Desktop](https://www.docker.com/products/docker-desktop)
+* Bash shell
+  * In Windows use [WSL](https://docs.microsoft.com/en-us/windows/wsl/install)
```

Comment:
It may be better to recommend for WSL2, as most of the folks here in team are using WSL2.
https://howto.lintel.in/wsl-vs-wsl-2-performance/
https://docs.microsoft.com/en-us/windows/wsl/compare-versions

Reply:
In the docs it is given 
>New Linux installations will be set to WSL 2 by default, but the wsl --set-version command can be used to downgrade from WSL 2 to WSL 1 as well. To see whether your Linux distribution is set to WSL 1 or WSL 2, use the command: wsl -l -v.


I think the default installation is wsl2 ?

Reply:
Yeah, as per docs that seems to be the case. I don't remember exact commands, but my default implementation was WSL1 when I had installed a month back. But, anyways, we have already used WSL2 in the documentation now.